### ISSUE TITLE
Checksum mismatch

### ISSUE THREAD

**Issue by lukesneeringer** [gertd/go-pluralize#8]
Having a weird issue trying to upgrade to the new go-pluralize.

Our Renovate PR that is trying to update it fails in CI with the following error:

```
verifying github.com/gertd/go-pluralize@v0.1.2: checksum mismatch
	downloaded: h1:3aljIkBHcMFPYdveAiihVWZ+7ZH2ALkRgJw0YPqqCrk=
	go.sum:     h1:8q9mYRD4K9X6fsldYCQKM3HZtNV/gfieh6xerucwoM4=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.
```

It has been doing it consistently for the last two weeks, so it does not seem like a one-off problem.

PR: https://github.com/googleapis/api-linter/pull/529
CI: [1](https://circleci.com/gh/googleapis/api-linter/6335?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link), [2](https://circleci.com/gh/googleapis/api-linter/6336?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

**Comment by RichardLindhout** [comment]
I decided to fork this repo to fix this issue: github.com/web-ridge/go-pluralize v0.1.5

**Comment by gertd** [comment]
Released update v0.1.6 sorry I did not see this earlier
